### PR TITLE
feat(orgbrowser): add support for StandardValueSet

### PR DESCRIPTION
**Pull request by sbresin**
### What does this PR do?
Adds support for retrieving StandardValueSet Metadata with the Org Browser.

There is no API call that lists all StandardValueSets, so i hardcoded the list of their names. The list comes from here: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/standardvalueset_names.htm

### What issues does this PR fix or reference?
#1579 

### Functionality Before
None: Currently it is not possible at all to retrieve StandardValueSets from the OrgBrowser.

### Functionality After
Screen Shot 2022-03-29 at 16 38 46

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts [hunk 1]
```diff
@@ -21,6 +21,89 @@ const validManageableStates = new Set([
   undefined // not part of a package
 ]);
 
+const STANDARDVALUESET_FULLNAME = 'StandardValueSet';
```
Comment (mohanraj-r) [inline packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts:24]: Can we reuse `standardValueSet.fullnames` from `source-deploy-retrieve` (`import { standardValueSet } from '@salesforce/source-deploy-retrieve/lib/src/registry'`) here instead

**Review by mohanraj-r — CHANGES_REQUESTED** [review]
Thanks for the contribution @sbresin 👍 

@RitamAgrawal Can you please take a look at this PR since you made changes to org browser recently

File: packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts (hunk 1 above)
Comment (sbresin) [reply packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts:25]: addressed.

thanks for checking it out this quick :)

**Comment by mohanraj-r** [comment]
Superseded by https://github.com/forcedotcom/salesforcedx-vscode/pull/3992

**Pull request closed without merging**